Atomic/atomic.py

```diff
@@ -30,6 +41,22 @@
 from docker.errors import NotFound
 
 IMAGES = []
+ATOMIC_LIBEXEC = os.environ.get('ATOMIC_LIBEXEC', '/usr/libexec/atomic')
+
+OSTREE_DOCKERIMG_PREFIX = "dockerimg/"
+SYSTEMD_UNIT_FILES_DEST = "/etc/systemd/system"
```

Comment:
Is this hard coded into ostree?  If I explode oci images onto ostree will it do it in dockerimage?  Just want to minimize the use of docker if possible.